server/src/testFixtures/java/io/crate/test/integration/CrateDummyClusterServiceUnitTest.java

```diff
@@ -142,4 +147,8 @@ protected ClusterService createClusterService(
         clusterService.start();
         return clusterService;
     }
+
+    static Rule.Context ruleContext(SQLExecutor e) {
+        return new Rule.Context(e.planStats(), CoordinatorTxnCtx.systemTransactionContext(), e.nodeCtx, UnaryOperator.identity());
+    }
```

Comment:
```suggestion
```

This is `e.ruleContext()` ?

Reply:
oh yes, thank you. This was a left-over.